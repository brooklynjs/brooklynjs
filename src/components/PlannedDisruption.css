.planned-disruption {
  border: 1px solid #FFF;
  font-size: 20px;
  font-weight: bold;
  margin: 60px 0;
  padding: 20px;
  text-align: center;
}

.planned-disruption span {
  display: block;
}
