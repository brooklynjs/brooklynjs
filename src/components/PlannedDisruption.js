import React from 'react';

import './PlannedDisruption.css';

import { longEventDate } from '../utils/time';

const PlannedDisruption = ({ eventDate }) => {
  return (
    <div className="planned-disruption">
      <span>PLANNED DISRUPTION</span>
      <span>{eventDate && longEventDate(eventDate)}</span>
      <br />
      <span>Please follow the directions below.</span>
    </div>
  );
};

export default PlannedDisruption;
