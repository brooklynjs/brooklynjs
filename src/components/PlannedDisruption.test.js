import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import PlannedDisruption from './PlannedDisruption';

it('renders', () => {
  const tree = ReactTestRenderer.create(
    <PlannedDisruption 
      eventDate={new Date(2017, 8, 21)} 
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
