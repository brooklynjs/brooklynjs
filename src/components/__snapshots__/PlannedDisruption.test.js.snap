// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="planned-disruption"
>
  <span>
    PLANNED DISRUPTION
  </span>
  <span>
    Thursday, September 21st, at 7pm
  </span>
  <br />
  <span>
    Please follow the directions below.
  </span>
</div>
`;
